src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CreatePullRequest.java

```diff
@@ -0,0 +1,35 @@
+package org.shipkit.internal.gradle.versionupgrade;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.shipkit.internal.util.GitHubApi;
+
+import java.io.IOException;
+
+class CreatePullRequest {
+
+    private static final Logger LOG = Logging.getLogger(CreatePullRequest.class);
+
+    public void createPullRequest(CreatePullRequestTask task) throws IOException {
+        createPullRequest(task, new GitHubApi(task.getGitHubApiUrl(), task.getAuthToken()));
+    }
+
+    public void createPullRequest(CreatePullRequestTask task, GitHubApi gitHubApi) throws IOException {
+        if(task.isDryRun()){
+            LOG.lifecycle("  Skipping pull request creation due to dryRun = true");
+            return;
+        }
+
+        LOG.lifecycle("  Creating a pull request of title '{}' in repository '{}' between base = '{}' and head = '{}'.",
+            task.getTitle(), task.getRepositoryUrl(), task.getBaseBranch(), task.getHeadBranch());
+
+        String body = "{" +
+            "  \"title\": \"" + task.getTitle() + "\"," +
+            "  \"body\": \"Please pull this in!\"," +
```

Comment:
Might be cool to write some more, something like: "This pull request was automatically created by Shipkit's "version-upgrade-customer" Gradle plugin (http://shipkit.org). Please merge it so that you are using fresh version of "xxx" dependency."